Number_Conversion/C++/Decimal_to_Binary.cpp

```diff
@@ -0,0 +1,37 @@
+// C++ program to convert numbers from decimal to binary form
+#include <iostream>
+#include <math.h>
+using namespace std;
+
+// Function to convert the number from decimal to binary form
+void printBinary(int n){
+
+  int size = log2(n) + 1;  // command to find out number of bits in the given number
```

Comment:
Please put the side comment before the line for which it is a comment e.g
// Array to store the binary number
int b[size], j = 0;